packages/dicom/python/itkwasm-dicom/README.md

```diff
@@ -12,3 +12,22 @@ Read files and images related to DICOM file format.
 ```sh
 pip install itkwasm-dicom
 ```
+
+## Acknowledgments
+
+`itkwasm-dicom` stands on the shoulders of the following giants:
+
+- [ITK](https://docs.itk.org/)
+- [GDCM](https://sourceforge.net/projects/gdcm/)
+- [DCMTK](https://dicom.offis.de/dcmtk.php.en)
+- [DCMQI](https://qiicr.gitbook.io/dcmqi-guide)
```

Comment:
@thewtex I would prefer to use the github link here: https://github.com/QIICR/dcmqi.